<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stakW</class>
 <widget class="QDialog" name="stakW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #6892d5;
color: #f8fcfb;</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #f8fcfb;</string>
   </property>
   <property name="text">
    <string>Book List</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="tech_radioButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#0091bf;
border-radius:10px;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Technical Books</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="all_radioButton_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#0091bf;
border-radius:10px;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>All Books</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="nonT_radioButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#0091bf;
border-radius:10px;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Non-Technical Books</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>140</y>
     <width>701</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #6892d5;
color: #f8fcfb;</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>21</y>
       <width>541</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 97, 81);</string>
    </property>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>541</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #6892d5;
color: #f8fcfb;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 240, 164);</string>
    </property>
    <widget class="QTableView" name="tableView_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>541</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #6892d5;
color: #f8fcfb;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
